python/ray/autoscaler/azure/defaults.yaml

```diff
@@ -118,6 +120,7 @@ setup_commands:
     # Note: if you're developing Ray, you probably want to create an AMI that
     # has your Ray repo pre-cloned. Then, you can replace the pip installs
     # below with a git checkout <your_sha> (and possibly a recompile).
+    - echo 'eval "$(conda shell.bash hook)"' >> ~/.bashrc
```

Comment:
What does this do?

Reply:
This initializes conda for bash shells / scripts and enables the conda environment to get loaded, otherwise conda activate fails and requires the user to manually activate.

Reply:
Ahh okay! Sounds good!